src/app/components/ScrollLock.tsx

```diff
@@ -51,13 +60,13 @@ const fadeIn = keyframes`
 
 interface OverlayProps extends HTMLAttributes<HTMLDivElement> {
   className?: string;
-  mobileOnly?: boolean;
+  mediumOnly?: boolean;
```

Comment:
Replaced because the overlay was serving no purpose on mobile and was just being hidden via z-index. Now the overlay is only used between mobile and mediumMobile breakpoints.